src/event_store_ms/src/main/java/events/HotelEvent.java

```diff
@@ -0,0 +1,21 @@
+/*package events;
```

Comment:
Czemu to jest zakomentowane? Jak jest nieużywane to wywalić.

Reply:
done